pwndbg/commands/leakfind.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Find a chain of leaks given some starting address.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+from queue import *
+
+import gdb
+
+import pwndbg.color.chain as C
+import pwndbg.color.memory as M
+import pwndbg.color.theme as theme
+import pwndbg.commands
+import pwndbg.vmmap
+
+config_arrow_right = theme.Parameter('chain-arrow-right', '—▸', 'right arrow of chain formatting')
+arrow_right = C.arrow(' %s ' % config_arrow_right)
+
+#Used to recursively print the pointer chain. 
+#addr is a pointer. It is taken to be a child pointer.
+#visitedMap is a map of children -> (parent,parent_start)
+def get_rec_addr_string(addr,visitedMap):
+    page = getPage(addr)
+    if not page is None:
+        if not addr in visitedMap:
+            return ""
+        
+        parentInfo = visitedMap[addr]
+        parent = parentInfo[0]
+        parent_base_addr = parentInfo[1]
+        if parent-parent_base_addr < 0:
+            curText = hex(parent_base_addr) + hex(parent-parent_base_addr)
+        else:
+            curText = hex(parent_base_addr) + "+"+ hex(parent-parent_base_addr)
+        if parent_base_addr == addr:
+            return ""
+        return get_rec_addr_string(parent_base_addr,visitedMap) + M.get(parent_base_addr,text=curText)+arrow_right
+    else:
+        return ""
+
+#Utility function to get a page from an address.
+def getPage(address):
+    return pwndbg.vmmap.find(address)
```

Comment:
This function is redundant (and function calls are expensive in Python). If we really want to use an alias, let's just do:
```suggestion
get_page = pwndbg.vmmap.find
```

Reply:
I just removed it altogether.